packages/next/src/client/components/react-dev-overlay/internal/container/StaticIndicator.tsx

```diff
@@ -1,23 +1,36 @@
+import * as React from 'react'
 import { Toast } from '../components/Toast'
 import { LightningBolt } from '../icons/LightningBolt'
 import { CloseIcon } from '../icons/CloseIcon'
 import type { Dispatcher } from '../../app/hot-reloader-client'
 
 export function StaticIndicator({ dispatcher }: { dispatcher?: Dispatcher }) {
   return (
-    <Toast className="nextjs-static-indicator-toast-wrapper">
-      <LightningBolt />
-      <span>Static route </span>
-      <button
-        onClick={() => {
-          dispatcher?.onStaticIndicator(false)
-        }}
-        className="nextjs-toast-hide-button"
-        aria-label="Hide static indicator"
-        style={{ marginLeft: 'var(--size-gap)' }}
-      >
-        <CloseIcon />
-      </button>
+    <Toast className={`nextjs-static-indicator-toast-wrapper`}>
+      <div className="nextjs-static-indicator-toast-icon">
+        <LightningBolt />
+      </div>
+      <div className="nextjs-static-indicator-toast-text">
+        Static route
+        <button
+          onClick={() => {
+            // When dismissed, we hide the indicator for 1 hour. Store the
+            // timestamp for when to show it again.
```

Comment:
This isn't obvious and likely leads to confused users. Why are we not getting this feedback for the Vercel toolbar? What are they doing differently that we're ok with having it there permanently?

At some point, we'll probably have dedicated Next.js devtools and then we probably don't want to hide an indicator permanently anyway.